### ISSUE TITLE
Indexes aren't automagically created

### ISSUE THREAD

**Issue by doismellburning** [logstash-plugins/logstash-output-elasticsearch#36]
(Apologies for the poor quality of this report; I'm unfamiliar with the details of the working behaviour)

With previous usage of the `elasticsearch_http` output in LS 1.4 and earlier, my ES indices were automatically created.

Using the `elasticsearch` output in LS 1.5.0-beta1, then on flush I get `IndexMissingException[[logstash-2015.01.01] missing]` - no index for me

(Related to #35)

**Comment by doismellburning** [comment]
My config was somewhat vanilla:

```
output {
    elasticsearch {
        host => redacted
        password => redacted
        port => 443
        protocol => "http"
        ssl => true
        user => redacted
    }
}
```

**Comment by suyograo** [comment]
should be fixed by #2

**Comment by dfaropennetwork** [comment]
Great thx
